test/dynamo/test_num_output.py

```diff
@@ -78,10 +78,14 @@ def do_test(self, model_class, expected_num_output):
         graph_input_tensor_ids,
         graph_input_xla_values,
     ) = torch_xla._XLAC._get_tensors_xla_device_data_node(outputs)
-
+    xla_args_tensor_ids = set(
+        tree_map_only(torch.Tensor,
+                      lambda input: torch_xla._XLAC._xla_get_tensor_id(input),
+                      inputs))
     graph_input_matcher = GraphInputMatcher(tensor_id_to_arg_idx,
                                             graph_input_tensor_ids,
-                                            graph_input_xla_values)
+                                            graph_input_xla_values,
+                                            xla_args_tensor_ids)
```

Comment:
can we add a test in following pr to check `graph_input_matcher. graph_input_xla_values`? Some of them should be None.

Reply:
Added a test for checking the None in the cached tensor list